tests/topotests/config_timing/test_config_timing.py

```diff
@@ -0,0 +1,186 @@
+#!/usr/bin/env python
+#
+# June 2 2021, Christian Hopps <chopps@labn.net>
+#
+# Copyright (c) 2021, LabN Consulting, L.L.C.
+# Copyright (c) 2019-2020 by
+# Donatas Abraitis <donatas.abraitis@gmail.com>
```

Comment:
I'm just not sure, why am I here? :smile:

Reply:
I copied some test that had your copyright, then rewrote most of it, but still you have a copyright claim b/c I started with your code (or code with your copyright on it) :)